tools/rum-models-generator/Sources/CodeGeneration/Print/SwiftPrinter.swift

```diff
@@ -157,9 +157,11 @@ public class SwiftPrinter: BasePrinter, CodePrinter {
     /// This will only be effective for types that support dynamically coded properties. If all properties are statically coded,
     /// the `Codable` implementation is generated by compiler.
     private func printExtensionsWithCodableImplementation(_ swiftStruct: SwiftStruct, parentSwiftStructs: [SwiftStruct] = []) throws {
-        let codedProperties = swiftStruct.properties.filter { $0.defaultValue == nil || $0.mutability != .immutable }
-        let staticallyCodedProperties = codedProperties.filter { $0.codingKey.isStatic }
-        let dynamicallyCodedProperties = codedProperties.filter { !$0.codingKey.isStatic }
+        let staticallyEncodedProperties = swiftStruct.properties.filter { $0.codingKey.isStatic }
+        let staticallyDecodedProperties = staticallyEncodedProperties.filter { !($0.defaultValue != nil && $0.mutability == .immutable) }
```

Comment:
Isn't it the same as `($0.defaultValue == nil || $0.mutability != .immutable)`? Which is a bit easier to read.

Reply:
✅ done